core/src/test/resources/WEB-INF/config-lucene.xml

```diff
@@ -125,6 +125,7 @@
       <field name="_valid_schematron-rules-iso" tagName="valid_schematron-rules-iso"/>
       <field name="_valid_schematron-rules-inspire" tagName="valid_schematron-rules-inspire"/>
       <field name="_valid_xsd" tagName="valid_xsd"/>
+      <field name="_valid_subtemplate" tagName="valid_subtemplate"/>
```

Comment:
the new fields `valid_subtemplate` is on top of `valid`, will have both, as we have `valid_xsd` for example. It is more precise if one day we have more types of validation for a subtemplate, but for now we have only one so it is the same.
The `valid` field is the one used in search result list anyway.

So should we remove it ?